frontend/src/components/pages/group.vue

```diff
@@ -310,7 +310,6 @@
             this.getGroup();
             analyticsServ.get(window.currentUrl, this.$route.meta.title, window.previousUrl);
         },
-
         destroyed(){
```

Comment:
Did you mean to add the updated link tracking here it's in the other single views (and list views) so it seems odd that this one was left out

Reply:
Added back! Along with one for the GroupCard.